framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -4304,7 +4304,7 @@ public long getEnergyFee(long timestamp) {
 
       return energyFee;
     } catch (Exception e) {
-      logger.error("getEnergyFee timestamp={} failed, error is {}", timestamp, e.getMessage());
+      logger.error("getEnergyFee timestamp={} failed, error is {}", timestamp, e.getMessage(), e);
```

Comment:
duplicate?

Reply:
Add the e will print out the stack trace, and the e.getMessage will not.Actually, the e.getMessage is for log analyze and stack trace will print in the next line